docs/ocean/design_docs/time-varying-wind.rst

```diff
@@ -0,0 +1,79 @@
+etting up time-varying forcing for MPAS-O
```

Comment:
```suggestion
setting up time-varying forcing for MPAS-O
```

Reply:
In general, I think your design doc would benefit from using proper Restructure Text format: http://docutils.sourceforge.net/docs/user/rst/quickstart.html

In particular, maybe a title and some formatting separate code from plain text.  You can see the result on GitHub by going to "view file" from the "files changed" tab.  Here's a link:
https://github.com/MPAS-Dev/MPAS-Model/blob/3f0a5a66ded2441070af090779870eccf8b10d72/docs/ocean/design_docs/time-varying-wind.rst

Reply:
Agreed

Reply:
For now it should be just structured in RST.  @sbrus89, do you want to take a crack at this?  I can help out here too since I've done this in the past.